LinkResolverService.cs

```diff
@@ -5,27 +5,28 @@
 using EPiServer.Web;
 using EPiServer.Web.Routing;
 using System;
+using System.Globalization;
 
 namespace Blend.Optimizely
 {
     [ServiceConfiguration]
     public class LinkResolverService
     {
-
         private Injected<IContentLoader> ContentLoader { get; }
+
         private Injected<ISiteDefinitionResolver> SiteDefinitionResolver { get; }
-        private Injected<IUrlResolver> UrlResolver { get; }
 
+        private Injected<IUrlResolver> UrlResolver { get; }
 
-        public virtual ResolvedLink? TryResolveLink(object? contentLink, LinkOptions options = LinkOptions.None)
+        public virtual ResolvedLink? TryResolveLink(object? contentLink, LinkOptions options = LinkOptions.None, string languageBranchId = "")
```

Comment:
I would recommend we keep the position of the language branch ID argument consistent. Right now in some methods it's second, and others it's third. I'd suggest keeping it as the third argument, since all resolve methods take the "item to be resolved" and the options arguments, but not all of them necessarily take a language ID argument. That would keep the first two consistent across all resolve methods.